src/components/NcGuestContent/NcGuestContent.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+ - @copyright Copyright (c) 2022 Christoph Wurst <christoph@winzerhof-wurst.at>
+ -
+ - @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ - @author Richard Steinmetz <richard@steinmetz.cloud>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+<docs>
+### General description
+
+This components provides a wrapper around a guest page content.
+
+### Usage
+
+```vue
+<template>
+	<NcGuestContent>
+		<h2>Hello guest</h2>
+		<span>How are you?</span>
+	</NcGuestContent>
+</template>
+```
+</docs>
+
+<template>
+	<div id="guest-content-vue">
+		<slot />
+	</div>
+</template>
+
+<script>
+/**
+ * Guest content container to be used for the guest content of your app
+ */
+export default {
+	name: 'NcGuestContent',
+}
+</script>
+
+<style lang="scss" scoped>
+#guest-content-vue {
+	color: var(--color-main-text);
+	background-color: var(--color-main-background);
+	min-width: 0;
+	border-radius: var(--border-radius-large);
+	box-shadow: 0 0 10px var(--color-box-shadow);
+	height: fit-content;
+	padding: 15px;
+	margin: 20px auto;
+}
+</style>
+
+<style lang="scss">
```

Comment:
This is not scoped and therefore breaks files app, settings, basically any page layout by missing the bottom border:
![grafik](https://user-images.githubusercontent.com/213943/205153457-c2bff5ee-f050-4d16-8802-637c4882ba7f.png)

![grafik](https://user-images.githubusercontent.com/213943/205153616-39427c40-3a91-4e7f-9435-99ead8a49a76.png)